order/interfaces.go

```diff
@@ -592,6 +634,10 @@ type Bid struct {
 	// happens out of band). This will only be used if the order version is
 	// VersionSidecarChannel or greater.
 	SidecarTicket *sidecar.Ticket
+
+	// UnannouncedChannels signals if the resulting channel needs to be
+	// announced or not.
+	UnannouncedChannels bool
```

Comment:
nit: shouldn't this be singular? Though we can be matched multiple times for the same bid so it could be plural. But then the comment should batch the name.